src/System.Text.Json/src/System/Text/Json/Node/JsonNodeOptions.cs

```diff
@@ -39,13 +39,17 @@ public JsonCommentHandling CommentHandling
         }
 
         /// <summary>
-        /// Gets or sets the maximum depth allowed when reading JSON, with the default (i.e. 0) indicating a max depth of 64.
+        ///   Gets or sets the maximum depth allowed when parsing JSON data,
+        ///   with the default (that is, 0) indicating a maximum depth of 64.
         /// </summary>
         /// <exception cref="ArgumentOutOfRangeException">
-        /// Thrown when the max depth is set to a negative value.
+        ///   The max depth is set to a negative value.
         /// </exception>
+        /// <value>
```

Comment:
@ahsonkhan If you build these changes, what do you see inside this file `corefx\artifacts\bin\System.Text.Json\netcoreapp-Debug\System.Text.Json.xml` ? Do you see a `<value>` field added somewhere?

Reply:
@ahsonkhan Ok so I found a small bug in my tool. It can fill some property `<value>` when the xml comment for a property has a `<returns>` (which is incorrect, but seems to happen sometimes). What it can't detect yet (and [I submitted a fix](https://github.com/carlossanlop/DocsPortingTool/commit/2e7bacbf60ebd9f1b4e3d53aba32f0db16aea306) for that) is detect real `<value>` items.
I'll run the tool again next week to see if I detect anything new in the Json namespaces.

Reply:
I think the value for IntelliSense is different than ECMAXML. For ECMAXML is value, for IntelliSense is returns, I don't think that VS automatically adds any return tag for properties when you do ///, so I consider them optional, but I try to add docs for them when I see empty ones.